geopandas/tests/test_extension_array.py

```diff
@@ -492,6 +492,12 @@ def test_argmin_argmax_empty_array(self):
     def test_argmin_argmax_all_na(self):
         pass
 
+    def test_shift_has_crs(self, data):
```

Comment:
Can you move this test to eg `test_pandas_methods.py` ? Because this file are only general tests that are inherited from pandas, so best to keep our own more specific tests separate.

Reply:
or `test_array.py`